package.json

```diff
@@ -71,19 +71,20 @@
     "lcov-result-merger": "^3.0.0",
     "lint-staged": "^8.1.5",
     "mariadb": "^2.0.3",
+    "markdownlint-cli": "^0.14.1",
```

Comment:
Should we add yet another tooling dependency?

I agree with linting pass you did in general but integrating it into the CI seems a little overkill 😛

Reply:
Well it gives us more tooling :smile: I was also thinking about a linting stage in CI so others build wont have to run lint everytime

Reply:
Yea that would seem reasonable, you can also squash TS and all of that in there.
The downside is that it will still try to run tests even if lint failed due to a coding issue (undefined variabled etc.).

Reply:
If one stage fails in travis all next stages are cancelled, so that should not be a problem

Reply:
Oh I misunderstood here, I thought you wanted to extract all lint tasks into a separate environment var gated pipeline. 😛